cardano-rosetta-server/src/server/services/network-service.ts

```diff
@@ -64,8 +80,8 @@ const configure = (
       peers: getPeersFromConfig(logger, topologyFile)
     };
   },
-  getExemptionTypes() {
-    return exemptionsFile ? JSON.parse(exemptionsFile) : [];
+  getExemptionTypes(logger: Logger) {
+    return getExemptionFile(logger);
```

Comment:
With the previous mentioned changes I think it can be reduced to this
```suggestion
    return process.env.EXEMPTION_TYPES_PATH ? exemptionsFile : [];
```

Reply:
Changed it as we discussed by slack